projects/HW102/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include "interrupt.h"
```

Comment:
Nitpick: the coding standards prescribe there should be a blank line between <standard> imports and "midsun-specific" imports.